dasharo-compatibility/dasharo-tools-suite.robot

```diff
@@ -0,0 +1,150 @@
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+...                     AND
+...                     Enable Network Boot
+...                     AND
+...                     Disable Secure Boot
+...                     AND
+...                     Upload And Mount DTS Flash ISO
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+
+
+*** Test Cases ***
+DTS001.001 Booting DTS from USB works correctly
+    [Documentation]    This test aims to verify that DTS is properly booting
+    ...    from USB.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS001.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS001.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    USB_emulated
+
+DTS002.001 DTS option Creating Dasharo HCL report works correctly
+    [Documentation]    This test aims to verify that the option Dasharo HCL
+    ...    report in the DTS menu properly creates the report.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS002.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS002.001 not supported
+    # Not supported due to SSH authorization error while sending the report
+    Skip
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Check HCL Report Creation
+
+DTS003.001 DTS option power-off DUT works correctly
+    [Documentation]    This test aims to verify that the option Power off
+    ...    system in the DTS menu turns off the DUT.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS003.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS003.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Sleep    3s
+    Write Into Terminal    10
+    Check Power Off In DTS
+
+DTS004.001 DTS option reboot DUT works correctly
+    [Documentation]    This test aims to verify that the option Reboot system
+    ...    in the DTS menu reboots the DUT.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS004.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS004.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    Sleep    3s
+    Write Into Terminal    11
+    Read From Terminal Until    ${TIANOCORE_STRING}
+
+DTS005.001 DTS drop-to-shell option works correctly
+    [Documentation]    This test aims to verify that Shell can be accessed via
+    ...    enabling SSH in the menu.
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    DTS005.001 not supported
+    Skip If    not ${DTS_SUPPORT}    DTS005.001 not supported
+
+    Power On
+    Boot Dasharo Tools Suite    iPXE
+    # Enable SSH in DTS menu
+    Write Into Terminal    8
```

Comment:
I do not get this test case. Option number 8 is about enablling SSH, while test says about drop to shell, which is option number 9.

Reply:
Now [rewritten](https://github.com/Dasharo/open-source-firmware-validation/pull/93/commits/65c06689a79e9ae794eb3e448280d3abfb7a8fef#diff-4d0e5cbb7f43565e1c6904f6f0c660951a6f60f7d187a787aa23d8fcab380777R75) as a more versatile test case